README.md

```diff
@@ -1,108 +1,100 @@
+Now create a new conda environment (replace `<environment-name>` with something descriptive, for example `mantid-mvesuvio`):
+- `mamba create -n <environment_name>`
 
-Open mantid workbench and navigate to the directory of the inputs file you copy-and-pasted in the last section.
+And activate the environment you created:
+- `mamba activate <environment_name>`
 
-**Warning**: Do not run the analysis routine on the default location for the inputs file. This will clutter the default `.mvesuvio` folder and may cause problems later down the line.
+Install the latest version of mantid workbench:
+- `mamba install mantidworkbench`
 
-Now you can alter the inputs of the file as you see fit. When you are finished, press run. This will run the routine and create a new directory for the output files inside the parent directory of the inputs file.
+Finally, install `mvesuvio` through the mantid channel:
+- `mamba install -c mantid/label/nightly mvesuvio`
 
-To run a new sample, simply copy-and-paste one of the inputs file into any directory you see fit, change the inputs and press run.
+You need to run the following command so that `mvesuvio` sets up some defaults:
+- `mvesuvio config`
 
-## Alternative Installation and Usage (Conda and CLI)
-We also provide an alternative conda installation for users wanting to use mantid workbench inside a conda environment.
+You can now start workbench by typing:
+- `workbench`
 
-### Install mamba
+### Quickstart (running first analysis)
 
-To use the `mvesuvio` package you will need to use the `conda` package manager (or preferably  `mamba`, a much faster implementation of `conda`).
+To run your very first analysis (and check that everything is working), go to your home folder and find the folder `.mvesuvio`.
+(The `.` in front of the directory name means this folder might be hidden by your OS, so you might have to turn on the option of showing hidden folders.
+)
 
-This is also the recommended best practice way of using the mantid packages.
+If the folder does not exist, try running `mvesuvio config` from the environment's terminal.
+Alternatively, you can open Mantid workbench, go the editor and run:
 
-To download and install mamba:
-- https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html
+```
+import mvesuvio as mv
+mv.set_config()
+```
 
-### Create and activate an environment for mantidworkbench and mvesuvio
+Once you have located the `.mvesuvio` folder, open Mantid workbench and inside it open the script `analysis_inputs.py` located inside `.mvesuvio`.
 
-Firstly create a conda environment where `mvesuvio` is going to be installed:
-- `mamba create -n <environment_name>`
+This script represents the basics for passing in the inputs of the analysis routine.
+Click the run button on the workbench to start the execution of the script.
+(Check that you have the archive search enabled, otherwise the Vesuvio runs will not be found).
```

Comment:
I would also mention setting the facility to ISIS and the instrument to Vesuvio in `Settings`.